src/CoreBundle/Service/MaintenanceModeServiceInterface.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace ChameleonSystem\CoreBundle\Service;
+
+use ChameleonSystem\CoreBundle\Exception\MaintenanceModeErrorException;
+
+interface MaintenanceModeServiceInterface
+{
+    /**
+     * Returns if the maintenance mode is activated (by file).
+     * Method should not query the database.
```

Comment:
The database is an implementation detail and should not be relevant in the interface. What's your intention behind this suggestion?

Reply:
Removed from interface for now: will be changed for #119